app/factories/access_code.py

```diff
@@ -21,7 +22,5 @@ class Meta:
     tickets_number = 30
     min_quantity = 10
     max_quantity = 20
-    marketer_id = 1
     valid_from = common.date_
     valid_till = common.dateEnd_
-    event_id = 1
```

Comment:
Revert these changes

Reply:
can revert in access code without any error. But if I do the same in Session factory the following error occurs.
![image](https://user-images.githubusercontent.com/39561007/73598939-0f106400-4564-11ea-803b-bdc95ab5140b.png)

Reply:
See how it is done in hook files